srcCpp/Infrastructure_common.h

```diff
@@ -14,42 +14,31 @@
   #include "Infrastructure_pro.h"
 #endif
 
+#ifdef RTI_VXWORKS
```

Comment:
I think we might not need this if we make nddsUtility:sleep() work properly in VxWorks (this belongs to the alarm issue though).

Reply:
Done